plugin/storage/integration/badgerstore_test.go

```diff
@@ -64,22 +63,7 @@ func (s *BadgerIntegrationStorage) initialize() error {
 	return nil
 }
 
-func (s *BadgerIntegrationStorage) clear() error {
```

Comment:
this function should remain and call factory.Close

Reply:
Updated the PR with the feedback.